src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -720,6 +720,18 @@ public void Initialize()
             LicenseManager.Initialize(_env, ContextPool);
             LatestVersionCheck.Instance.Check(this);
 
+            if (Configuration.Backup.MaxNumberOfConcurrentBackups != null)
+            {
+                MaxNumberOfConcurrentBackups = Configuration.Backup.MaxNumberOfConcurrentBackups.Value;
+            }
+            else
+            {
+                LicenseManager.GetCoresLimitForNode(out var utilizedCores);
+                MaxNumberOfConcurrentBackups = Math.Max(1, utilizedCores / 2);
+            }
+
+            ConcurrentBackupsSemaphore = new SemaphoreSlim(MaxNumberOfConcurrentBackups);
```

Comment:
This is not being refreshed after assigned cores is changed or license is switched

Reply:
Done